src/Microsoft.AspNetCore.Sockets.Common/Microsoft.AspNetCore.Sockets.Common.csproj

```diff
@@ -8,6 +8,7 @@
     <NoWarn>$(NoWarn);CS1591</NoWarn>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;signalr</PackageTags>
+    <RootNamespace>Microsoft.AspNetCore.Sockets</RootNamespace>
```

Comment:
We don't usually put things in the actual `.Common` namespace when we make a `.Common` project.